setup.h

```diff
@@ -49,6 +49,9 @@ class TeletextSetup {
    cString txtFontName;
    cStringList txtFontNames;
    int txtFontIndex;
+   int txtG0Block;
+   int txtG2Block;
+   const char *txtBlock[16];
```

Comment:
This is not part of the actual setup. Just define this helper as local variable in the setup menu method when used.